app/models/manageiq/providers/ansible_tower/shared/automation_manager/amazon_credential.rb

```diff
@@ -23,4 +23,10 @@ module ManageIQ::Providers::AnsibleTower::Shared::AutomationManager::AmazonCrede
   }.freeze
 
   API_ATTRIBUTES = COMMON_ATTRIBUTES.merge(EXTRA_ATTRIBUTES).freeze
+
+  API_OPTIONS = {
+    :type       => N_('cloud'),
```

Comment:
I think `:type` here doesn't need to be inside `N_()`. It's not going to be presented in the UI as a nice string, correct? Just the label should be enough here.